Patch Image

maxmii opened the pull request:
Update packages for libcrypto3 and libssl3 to fix vulnerabilities from trivy scan

Commit:
Update packages for libcrypto3 and libssl3 to fix vulnerabilities from Trivy scan

ianrelf reviewed (commented):
Are you sure you want to remove `CMD []`?

maxmii committed:
Update dockerfile add line

Re add the command CMD [ "redis-server", "/usr/local/etc/redis/redis.conf"]. based on Ian's comments

maxmii commented:
Updated code based on @ianrelf comments

s-arunachalam reviewed (approved):
ok
